### PR TITLE
Add acknowledgements

### PR DESCRIPTION
**Pull request by GuillaumeDesforges**

### CODE REVIEW

File: styles/pages/index.less [hunk 1]
```diff
@@ -95,7 +95,6 @@
   }
 }
 
-
```
Comment (Radvendii) [inline styles/pages/index.less:98]: Is this just removing a newline? In my expeirence style adjustments usually go in their own commit / PR.

File: styles/pages/index.less [hunk 2]
```diff
@@ -135,7 +134,6 @@ section.faq {
   .img {
     display: none;
   }
-
```
Comment (Radvendii) [inline styles/pages/index.less:138]: Is this just removing a newline? In my expeirence style adjustments usually go in their own commit / PR.

File: index.html [hunk 3]
```diff
@@ -293,3 +293,27 @@ <h2>FAQ</h2>
...
...zdDAiIHdpZHRoPSIxNS44IiBoZWlnaHQ9IjMuOCIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik02NC4zLDIyLjdMNjQuMywyMi43aDE1LjFWMTdINjQuM2MtMSwwLTEuOS0wLjgtMS45LTEuOXYtMS45VjUuN2gtMy44Yy0xLDAtMS45LDAuOC0xLjksMS45djUuN3YxLjlsMCwwDQoJCWMwLDEsMC44LDEuOSwxLjksMS45aDAuNWgxLjRjMSwwLDEuOSwwLjgsMS45LDEuOXYxLjlDNjIuNCwyMS44LDYzLjIsMjIuNyw2NC4zLDIyLjdMNjQuMywyMi43eiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik03OS40LDBINjQuM2MtMSwwLTEuOSwwLjgtMS45LDEuOXYzLjhoMTdDNzkuNCw1LjcsNzkuNCwwLDc5LjQsMHoiLz4NCgk8cG9seWdvbiBjbGFzcz0ic3QwIiBwb2ludHM9Ijc2LjUsMTMuMiA2MC40LDEzLjIgNjAuNCw5LjQgNzYuNSw5LjQgCSIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xMTUuMywyMi43TDExNS4zLDIyLjdoMTAuNGMxLDAsMS45LTAuOCwxLjktMS45VjE3aC0xMi4zYy0xLDAtMS45LTAuOC0xLjktMS45di0xLjlWNS43aC0zLjgNCgkJYy0xLDAtMS45LDAuOC0xLjksMS45djUuN3YxLjlsMCwwYzAsMSwwLjgsMS45LDEuOSwxLjloMC41aDEuNGMxLDAsMS45LDAuOCwxLjksMS45djEuOUMxMTMuNCwyMS44LDExNC4yLDIyLjcsMTE1LjMsMjIuNw0KCQlMMTE1LjMsMjIuN3oiLz4NCgk8cGF0aCBjbGFzcz0ic3QwIiBkPSJNMTMwLjQsMGgtMTUuMWMtMSwwLTEuOSwwLjgtMS45LDEuOXYzLjhoMTdMMTMwLjQsMEwxMzAuNCwweiIvPg0KCTxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xMjcuNiwxMy4yaC04LjVWOS40aDEyLjNjMSwwLDEuOSwwLjksMS45LDEuOXYxLjlWMTdoLTUuN0MxMjcuNiwxNywxMjcuNiwxMy4yLDEyNy42LDEzLjJ6Ii8+DQo8L2c+DQo8L3N2Zz4NCg=="/>
```
Comment (Radvendii) [inline index.html:315]: Is there a reason these are inlined, as opposed to being stored as an image file?

**Review by Radvendii — COMMENTED** [review]
One additional comment: I'm not sure if it's worth tweaking (either through css styling or photoshopping images), but a few of the logos have their own internal padding, which makes the space between logos slightly (but noticeably) uneven.

Otherwise looks good. I checked that it scales well on different device sizes, too.

File: styles/pages/index.less (hunk 1 above)
Comment (GuillaumeDesforges) [reply styles/pages/index.less:98]: A bit nitpicking in this context imo, but I agree it's best practice in general to make formatting commits independently to code changes

File: index.html (hunk 3 above)
Comment (GuillaumeDesforges) [reply index.html:315]: No specific reason, I'll update

File: styles/pages/index.less (hunk 1 above)
Comment (Radvendii) [reply styles/pages/index.less:98]: Agreed that it's nitpicking XD. Thanks for putting up with it.

**Review by Radvendii — APPROVED** [review]
Looks good to me.
